### DISCUSSION TITLE
Verifying markdown file in a specific format

### DISCUSSION THREAD

**Discussion by taras** [textlint/textlint#1422] [Q&A]
I want to verify that a markdown has specific structure. For example, 

```md
## <directory_name>

<description>
```

Is there a way to do this with textlint?

**Comment by azu** [comment]
I do not currently maintain it, but the following rule does something close.

- https://github.com/azu/textlint-rule-no-empty-section

The mechanism is to traverse TxtAST to check if the structure is Header > Description.

- https://textlint.github.io/docs/txtnode.html
